Apps/HealthGateway.sln

```diff
@@ -0,0 +1,219 @@
+﻿

```

Comment:
I am guessing this is only necessary for visual studio. Will it affect sonarqube reports? SQ tends to be quite aggressive when looking for projects

Reply:
only visual studio, also the builds happen for the individual solutions we have in each app folder so it doesn't impact sonar qube